chef/cookbooks/cpe_utils/libraries/node_functions.rb

```diff
@@ -273,6 +273,18 @@ def windows?
       return node['os'] == 'windows'
     end
 
+    def parallels?
+      return virtual_macos_type == 'parallels'
```

Comment:
you dont need the return. there is an im[plict return. 
`virtual_macos_type == 'parallels'`